core/mobile/src/backend/MobileRpcServer.ts

```diff
@@ -164,6 +164,21 @@ export function setupMobileRpc() {
 
     server = new MobileRpcServer();
     clearInterval(retainUvLoop);
+    retainUvLoop = undefined;
+  });
+
+  MobileHost.onWillTerminate.addListener(() => {
+    if (typeof (retainUvLoop) !== "undefined") {
```

Comment:
Unless I'm missing something, this should be:

```ts
if (retainUvLoop !== undefined)
```

Reply:
I'm pretty sure it has to be that way. There is a weird case where undefined could be a different value. However, typeof is part of the language and is thus more reliable.